releng/tools/_get_rust_executables.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import shutil
+import json
+import argparse
+from typing import TextIO, List
+
+
+def dir_path(path: str) -> str:
+    if not os.path.isdir(path):
+        raise NotADirectoryError(path)
+    return path
+
+
+def get_rust_executables(json_input: TextIO) -> List[str]:
+    executables: list[str] = []
+
+    for line in json_input:
+        artifact = json.loads(line)
+        if artifact['reason'] != 'compiler-artifact':
+            continue
+        executable = artifact['executable']
+        if executable is None:
+            continue
+        if type(executable) is not str:
+            raise TypeError("executable in json has to be a string")
+        executables.append(executable)
+
+    return executables
+
+
+def get_filename(filepath: str, prefix: str = '') -> str:
+    assert os.path.isfile(filepath)
+    return prefix + os.path.basename(filepath)
+
+
+def copy_executables(source: TextIO, dest_dir: str, prefix: str = ''):
```

Comment:
Vidím, že tento command se už nepoužívá; otázkou je jestli není lepší místo kopie dělat symlink (asi netřeba měnit).

Reply:
Možná... idea původně byla ať v té složce je vyloženě spustitelný soubor (co se dá třeba i přesunout/cokoliv) - ale se tam přidá víc těch souborů, tak to nefunguje...